src/k8s-extension/azext_k8s_extension/consts.py

```diff
@@ -21,7 +21,7 @@
 PROVISIONED_CLUSTER_TYPE = "provisionedclusters"
 
 CONNECTED_CLUSTER_API_VERSION = "2021-10-01"
-MANAGED_CLUSTER_API_VERSION = "2021-10-01"
+MANAGED_CLUSTER_API_VERSION = "2022-11-01"

```

Comment:
This is the api-version for AKS calls - wanted to confirm that your intention is to update this!

Reply:
@Arif-lakhani - can you please double check on this? From Narayan's comment above, looks like we use this API  version to talk to AKS RP.